frontend/src/Pages/BMLogHoursPage/BMLogHoursPage.js

```diff
@@ -0,0 +1,120 @@
+        }
+        else if (startDT > endDT) {
+            alert("Start and end dates are invalid. Please enter dates again.");
+            return false;
+        }
+        return true;
+    }
+
+    //push form as element in opportunity array in volunteer schema
+    function handleSubmit() {
+        // make sure opportunties is defined
+        if (opportunities === null || opportunities === undefined) return;
+        //check if form is valid
+        if (!isValidForm()) return;
+        
+        //if opportunities alreayd has the task, append start and end d/t to existing array
+        var contains = false;
+        opportunities.forEach(function(item) {
+            if(item.task.toLowerCase() === tasks.toLowerCase()) {
+                item.start.push(startDT);
+                item.end.push(endDT);
+                contains = true;
+            }
+        })
+        //else add the opportunity
+        if(!contains) {
+            var newOpp = {
+                task: tasks,
+                start: [startDT],
+                end: [endDT],
+                description: "",
+                donated: []
+            }
+    
+            opportunities.push(newOpp)
+        }
+
+        // push new opportunities to database
+        var newOpportunites = props.user.opportunities;
+        console.log(newOpportunites);
+        newOpportunites[key] = opportunities;
+        const updateVolunteer = {
+            _id: props.user._id,
+            AOI: props.user.AOI,
+            address: props.user.address,
+            boardMember: props.user.boardMember,
+            communityRole: props.user.communityRole,
+            email: props.user.email,
+            experience: props.user.experience,
+            firstName: props.user.firstName,
+            googleId: props.user.googleId,
+            lastName: props.user.lastName,
+            outreach: props.user.outreach,
+            phoneNum: props.user.phoneNum,
+            signature: props.user.signature,
+            tasks: props.user.tasks,
+            username: props.user.username,
+            workHistory: props.user.workHistory,
+            _v: props.user._v,
+            opportunities: newOpportunites
+        }
+        fetch(`${process.env.REACT_APP_SERVER_URL}/api/postVolunteer`, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(updateVolunteer)
+        }).then(response => {
+            response.json().then(data => {
+                console.log("Successful" + data);
+            });
+        });
+    }
+
+    return (
+        <body id="logHoursPage">
+            <h1 id="LHPageTitle">BOARD MEMBER LOG HOURS</h1>
+            <Form >
+                <label for="startDateTime">Start Date and Time:</label><br />
+                <input type="datetime-local" id="startDate" name="startDateTime" onChange={e => setStartDT(e.target.value)} /><br /><br />
```

Comment:
Chome makes this a date picker but firefox does not, so try to use these date pickers if possible:

 import { MuiPickersUtilsProvider } from '@material-ui/pickers';
import DateMomentUtils from '@date-io/moment'
import DateFnsUtils from '@date-io/date-fns'
import {KeyboardDateTimePicker} from '@material-ui/pickers';

There's an example on the opportunityDetail page